lib/src/widgets/send_message_widget.dart

```diff
@@ -0,0 +1,258 @@
+                    ),
+                  ),
+                  Padding(
+                    padding: EdgeInsets.fromLTRB(6, 6, 6, _bottomPadding),
+                    child: Stack(
+                      alignment: Alignment.bottomCenter,
+                      children: [
+                        if (replyMessage.message.isNotEmpty)
+                          Container(
+                            decoration: BoxDecoration(
+                              color: widget.sendMessageConfig
+                                      ?.textFieldBackgroundColor ??
+                                  Colors.white,
+                              borderRadius: const BorderRadius.vertical(
+                                  top: Radius.circular(14)),
+                            ),
+                            margin: const EdgeInsets.only(
+                              bottom: 17,
+                              right: 0.4,
+                              left: 0.4,
+                            ),
+                            padding: const EdgeInsets.fromLTRB(9, 9, 9, 30),
+                            child: Container(
+                              margin: const EdgeInsets.only(bottom: 2),
+                              padding: const EdgeInsets.symmetric(
+                                vertical: 4,
+                                horizontal: 6,
+                              ),
+                              decoration: BoxDecoration(
+                                color: widget
+                                        .sendMessageConfig?.replyDialogColor ??
+                                    Colors.grey.shade200,
+                                borderRadius: BorderRadius.circular(12),
+                              ),
+                              child: Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                mainAxisSize: MainAxisSize.min,
+                                children: [
+                                  Row(
+                                    mainAxisAlignment:
+                                        MainAxisAlignment.spaceBetween,
+                                    children: [
+                                      Text(
+                                        "${PackageStrings.replyTo} $replyTo",
+                                        style: TextStyle(
+                                          color: widget.sendMessageConfig
+                                                  ?.replyTitleColor ??
+                                              Colors.deepPurple,
+                                          fontWeight: FontWeight.bold,
+                                          letterSpacing: 0.25,
+                                        ),
+                                      ),
+                                      IconButton(
+                                        constraints: const BoxConstraints(),
```

Comment:
if not required, remove it

Reply:
Added this to remove IconButton internal padding